tests/transform.spec.ts

```diff
@@ -212,8 +235,20 @@ describe('transform tests', () => {
     )
 
     expect(transformCode(options('import { TestBase } from "./test";')).content).toEqual(
+      "import { TestBase } from './test';\n"
+    )
```

Comment:
Git is a little confused here because of the added tests, but this is the diff for the changed test:
```diff
-    expect(transformCode(options('import { TestBase } from "./test";')).content).toEqual(
-      "import { TestBase } from './utils/test';\n"
-    )
+    expect(transformCode(options('import { TestBase } from "./test";')).content).toEqual(
+      "import { TestBase } from './test';\n"
+    )
```

Essentially, relative paths should stay relative and I had previously made this test incorrect.